source/includes/steps-create-data-encryption-key.yaml

```diff
@@ -77,6 +99,33 @@ content: |
              .build();
         
            ClientEncryption clientEncryption = ClientEncryptions.create(clientEncryptionSettings);
+     .. tab::
+        :tabid: nodejs
+
+        .. code-block:: javascript
+
+            const connectionString = "mongodb://localhost:27017";
+            const keyVaultNamespace = "encryption.__keyVault";
+            const client = new MongoClient(process.env.MONGO_URL, {
+                useNewUrlParser: true,
+                useUnifiedTopology: true
+            });
+
+            client.connect()
+            .then((clientConnection)=>{
```

Comment:
Can we move this `.then` to the previous line?

```javascript
client.connect().then(clientConnection => {
  const encryption = new ClientEncryption(client, {
    keyVaultNamespace,
    kmsProviders
  });
  ...
})
```

Reply:
Fixed